app/controllers/discovered_hosts_controller.rb

```diff
@@ -184,16 +184,22 @@ def auto_provision_all
 
   private
 
+  def remove_slashes(string)
+    string.slice!(0)
+    string.slice!(string.length-1)
+    string
+  end
```

Comment:
Ok don't remove slashes, the idea is to accept plain regexp. For instance, when doing

   egrep ^Line

you also don't provide these.